HandyImage.user.js

```diff
@@ -1135,12 +1136,11 @@ function makeworld()
 		i = q('img[data-width]');
 		break;
 	case "22pixx.xyz":
-		if(window.location.href.indexOf("/ia-i/") != -1)
+	        var img = window.location.href.match(/i[ab]?-([a-z].+(?:\.jpe?g|png|gif|webp))/i);
+		if(img)
 		{
-			window.location.href = window.location.href.replace('/ia-i/', '/ib-i/');
-			break;
+		   i = {src : window.location.origin + "/" + img[1]};
```

Comment:
are you sure that line is right and working?

Reply:
Yeah, i tested